vector/src/main/java/im/vector/app/core/pushers/VectorPushHandler.kt

```diff
@@ -168,14 +111,16 @@ class VectorMessagingReceiver : MessagingReceiver() {
      *
      * @param pushData Object containing message data.
      */
-    private fun onMessageReceivedInternal(pushData: PushData) {
+    private suspend fun handleInternal(pushData: PushData) {
         try {
+            // tchap : use BuildCongig instead of BuildMeta
```

Comment:
idem: I think this comment should be a TODO. It looks like a Tchap decision to keep BuildConfig, which is not true. This is just out of the scope here

"Tchap TODO: use BuildMeta instead of BuildConfig"